iceoryx_posh/source/popo/ports/server_port_data.cpp

```diff
@@ -21,15 +21,28 @@ namespace iox
 {
 namespace popo
 {
+/// @todo iox-#27 remove when the port queue policies are consolidated
```

Comment:
Since this PR is already quite big, I didn't want to do this cleanup now since it will affect quite many files. If you agree, I will create a follow up PR while this PR is still in review

Reply:
Corresponding PR is #1075